.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/sdl/SDL2_image"]
 	path = lib/sdl/SDL2_image
 	url = https://github.com/libsdl-org/SDL_image.git
+[submodule "lib/usb/ohci_usbhostlib"]
```

Comment:
*(This is a generic comment about PR description, but made on code so we can resolve it more easily)*

> Some very useful features for developers is support for CDC class USB-Serial adaptors for improved 'printf debugging'.

I think this deserves a sample and a replacement hal/debug.h in a follow-up.